README.md

```diff
@@ -1,45 +1,29 @@
+[![CircleCI](https://img.shields.io/circleci/build/github/xpring-eng/xpring-common-js/master?style=flat-square&token=0ed9e0790d44d163a5bf2793724fc85d98c3845b)](https://circleci.com/gh/xpring-eng/xpring-common-js/tree/master) [![CodeCov](https://img.shields.io/codecov/c/github/xpring-eng/xpring-common-js/master?style=flat-square&token=08b799e2895a4dd6add40c4621880c1a)]((https://codecov.io/gh/xpring-eng/xpring-common-js))
 [![CircleCI](https://img.shields.io/circleci/build/github/xpring-eng/xpring-common-js/master?style=flat-square&token=0ed9e0790d44d163a5bf2793724fc85d98c3845b)](https://circleci.com/gh/xpring-eng/xpring-common-js/tree/master) [![CodeCob](https://img.shields.io/codecov/c/github/xpring-eng/xpring-common-js/master?style=flat-square&token=08b799e2895a4dd6add40c4621880c1a)]((https://codecov.io/gh/xpring-eng/xpring-common-js))
 
-# Xpring Common JS
-Xpring Common JS provides JavaScript based common functionality to all client side libraries in the Xpring Platform. 
+# Xpring Common JavaScipt
 
-Developers who want to access the Xpring platform should use one of the following top-level libraries:
-- [XpringJS](http://github.com/xpring-eng/xpring-js): A JavaScript interface for the Xpring Platform
-- [XpringKit](http://github.com/xpring-eng/xpringkit): A Swift interface for the Xpring Platform
+Xpring Common JavaScript Library provides common JavaScript functionality to all client side libraries in [Xpring SDK](https://github.com/xpring-eng/xpring-sdk). This library is built to be consumed as a dependency to other libraries, rather than a standalone library.
```

Comment:
```suggestion
Xpring Common JavaScript Library provides common JavaScript functionality to all client-side libraries in [Xpring SDK](https://github.com/xpring-eng/xpring-sdk). This library is built to be consumed as a dependency to other libraries, rather than as a standalone library.
```